src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Class Lottery does not have to inherit any attributes or methods of ```ColorSupplier``` class, so no need to extend ```Lottery``` from ```ColorSupplier```